### PR TITLE
Add logic to support EventHub config

### PR DESCRIPTION
**Pull request by yantang-msft**
Add logics so user can enable MA to EventHub from powershell

### CODE REVIEW

**Comment by markcowl** [comment]
tests?

**Comment by yantang-msft** [comment]
Hi Mark,

IIRC, today is the Code Complete date for powershell? What’s the exact time?
I’m adding test cases, but we want to get this change in for the April release,  so we don’t want to miss the last chance of merging.

Thanks!
Yanming

**Comment by markcowl** [comment]
@yantang-msft if you guys sign off on the functionality, it is ok to add the tests later this week.  But please file an issue for this in GitHub and refer to it here.

**Comment by markcowl** [comment]
on demand run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/618/

File: src/ServiceManagement/Compute/Commands.ServiceManagement.Test/UnitTests/Cmdlets/IaaS/Extensions/Diagnostics/DiagnosticsHelperTest.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using Microsoft.WindowsAzure.Commands.ServiceManagement.Common;
+using Xunit;
+
+namespace Microsoft.WindowsAzure.Commands.ServiceManagement.Test.UnitTests.Cmdlets.IaaS.Extensions.Diagnostics
+{
+    public class DiagnosticsHelperTest
+    {
+        [Fact]
```
Comment (markcowl) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/UnitTests/Cmdlets/IaaS/Extensions/Diagnostics/DiagnosticsHelperTest.cs]: These should be attributed as check-in tests

Comment (yantang-msft) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/UnitTests/Cmdlets/IaaS/Extensions/Diagnostics/DiagnosticsHelperTest.cs]: I only see BVT defined in this project which looks closer.
Here is what marked for DSC extension, [Trait(Category.Functional, Category.BVT)]. Should I follow this attribute or add "CheckIn" in the category?

Comment (yantang-msft) [inline src/ServiceManagement/Compute/Commands.ServiceManagement.Test/UnitTests/Cmdlets/IaaS/Extensions/Diagnostics/DiagnosticsHelperTest.cs]: @markcowl Attribute added. See if it's correct.

**Comment by markcowl** [comment]
on demand run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/625/

**Comment by yantang-msft** [comment]
@markcowl The failed test case is irrelevant to this change, it will also fail before any of my commits.
However, I tested it will pass with current dev branch (and after merging my changes as well). So I assume there is some test framework change which fixed this issue, it's just that I branched before this fix.

**Comment by hovsepm** [comment]
new demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/628/

**Pull request merged**